beacon_node/beacon_chain/src/validator_monitor.rs

```diff
@@ -37,6 +41,18 @@ pub const HISTORIC_EPOCHS: usize = 10;
 /// Prometheus cardinality and log volume.
 pub const DEFAULT_INDIVIDUAL_TRACKING_THRESHOLD: usize = 64;
 
+/// This is a variable used in an experimental feature, as per:
+/// https://github.com/sigp/lighthouse/issues/4526
```

Comment:
```suggestion
```

I don't think we need to call the feature "experimental", once it's reviewed and merged it's a legit feature!

It might seem like a nitpick, but we generally avoid adding "experimental" features. We try to make all features production ready, without the need for qualification. It's not always the case, but it's how we like to operate.